docs/config.rst

```diff
@@ -262,20 +262,26 @@ This option takes a number from 0.0 to 1.0, representing the percent of
 events to allow through to server (from 0% to 100%). By default all
 events will be sent to the Sentry server.
 
-Inapp Classes
-~~~~~~~~~~~~~
+"In Application" Stack Frames
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Sentry differentiates stack frames that are directly related to your application
+("in application") from stack frames that come from other packages. The difference
```

Comment:
It'd be helpful to clarify "other packages" here, (e.g. "other packages such the standard library, frameworks, or other dependencies.")

It might be worth also noting the difference between this behavior and the previous one, at least for a little bit. Your call here.